src/PhpDoc/PhpDocBlock.php

```diff
@@ -129,7 +129,7 @@ private static function resolvePhpDocBlock(
 		if (
 			(
 				$docComment === null
-				|| preg_match('#\{@inheritdoc\}#i', $docComment) > 0
+				|| preg_match('#[\s\{]@inheritdoc[\s\}]#i', $docComment) > 0
```

Comment:
Hi, I don't understand why the \s around? Looks like a BC break.

Reply:
Me neither, why not just `[{]?`?

Reply:
Maybe just even `{?`? (not a regex expert)

Reply:
@ondrejmirtes Yeah that works too.

![image](https://user-images.githubusercontent.com/1752683/54345310-1aecf200-4643-11e9-8b0b-c40f48c0a8c3.png)

The `?` operator is lazy.

Reply:
Although you do of course need the backslash (`\{?`) then.

Reply:
Actually, we probably just want:

```regex
#(@inheritdoc)|(\{@inheritdoc\})#i
```

We don't want to match `{@inheritdoc` and `@inheritdoc}`.

Reply:
Yes, I realized that too. My next possible solution is `#{@inheritdoc}|@inheritdoc#` - what do you think about it?

Reply:
I just realized it doesn't really make a difference. If you enter `{@inheritdoc` the `@inheritdoc` check will still succeed.

Reply:
Ok, less sleep than I expected...
I will change it to @iluuu1994 variant then, as we both suggested.